pkg/rpc/client/client_test.go

```diff
@@ -71,7 +71,7 @@ func TestScanner_Scan(t *testing.T) {
 								Title:            "DoS",
 								Description:      "Denial os Service",
```

Comment:
I suspect this should be:
```suggestion
								Description:      "Denial of Service",
```

My tooling only looks at 3+ letter words by default (2 letter words had too much noise).